arwen/cryptoapi/cryptoei.go

```diff
@@ -383,6 +494,49 @@ func v1_4_verifyBLS(
 	return 0
 }
 
+//export v1_4_managedVerifyBLS
+func v1_4_managedVerifyBLS(
+	context unsafe.Pointer,
+	keyHandle int32,
+	messageHandle int32,
+	sigHandle int32,
+) int32 {
+	runtime := arwen.GetRuntimeContext(context)
+	crypto := arwen.GetCryptoContext(context)
+	metering := arwen.GetMeteringContext(context)
+	managedType := arwen.GetManagedTypesContext(context)
+	metering.StartGasTracing(verifyBLSName)
+
+	gasToUse := metering.GasSchedule().CryptoAPICost.VerifyBLS
+	metering.UseAndTraceGas(gasToUse)
+
+	keyBytes, err := managedType.GetBytes(keyHandle)
+	if arwen.WithFault(err, context, runtime.ManagedBufferAPIErrorShouldFailExecution()) {
```

Comment:
`WithFaultIfFailAlwaysActive()`? Not sure right now, might be ok as is.

Reply:
everyone is returning true. all errors return true.